jmix-translations/content/io/jmix/flowui/messages.properties

```diff
@@ -83,6 +83,7 @@ validationFail.defaultRequiredMessage=The field is required
 validationFail.defaultRequiredMessageWithFieldLabel=The '%s' field is required
 
 validation.defaultMessage = Incorrect value of field '%s'
+validation.defaultMessageForUnparseableValue = Incorrect string entered for field value
```

Comment:
I don't like 'string', because for the `BigDecimalField`, users enter `BigDecimal` value, not a string (in fact, whey are restricted to enter letters) and the message may be confusing.

Let it be:  `Unparseable value` / `Неправильный формат значения`